<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>603</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KGminer Control Panel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="processFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="filterpickerB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>FilterPicker</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="xcorrB">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cross
Correlation</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="tempB">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Temp</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="highpass1B">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Highpass Filter</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="highpass2B">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Highpass Filter</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>80</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>150</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>200</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="binderB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Binder</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loc1B">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>NonLinLoc</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="datarecieverB">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Data Reciever</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startEWButton">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start All</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopEWButton">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stop All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="loc2B">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>NonLinLoc</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loc3B">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>NonLinLoc</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="statusFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>371</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="functionFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="swarmButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>81</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Waveform
Viewer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eventViewerButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>5</y>
       <width>81</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Event
Viewer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dataExtractorButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>5</y>
       <width>81</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Event
Maker</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stainfoButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>81</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Station Info.
Viewer</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="incomingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Feeding</string>
    </property>
   </widget>
   <widget class="QLabel" name="processLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Process Layout</string>
    </property>
   </widget>
   <widget class="QLabel" name="pickLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pick Status</string>
    </property>
   </widget>
   <widget class="QFrame" name="pickframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>371</width>
      <height>270</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="actionLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionKorean"/>
    </widget>
    <widget class="QMenu" name="actionStation">
     <property name="title">
      <string>Config stations Info.</string>
     </property>
     <addaction name="actionConfigStaInfo"/>
     <addaction name="actionLoadStaFile"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionStation"/>
    <addaction name="separator"/>
    <addaction name="actionImportData"/>
    <addaction name="actionDataMonitor"/>
    <addaction name="separator"/>
    <addaction name="actionLanguage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_KGminer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionKorean">
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="menuNewStaInfo">
   <property name="text">
    <string>Config new stations Info.</string>
   </property>
  </action>
  <action name="menuLoadStaFile">
   <property name="text">
    <string>Load a station file</string>
   </property>
  </action>
  <action name="menuImportTank">
   <property name="text">
    <string>Import a tank file</string>
   </property>
  </action>
  <action name="actionConfigStaInfo">
   <property name="text">
    <string>Config new stations Info.</string>
   </property>
  </action>
  <action name="actionLoadStaFile">
   <property name="text">
    <string>Load a station file</string>
   </property>
  </action>
  <action name="menuMakeTank">
   <property name="text">
    <string>Make a tank file</string>
   </property>
  </action>
  <action name="actionAdd_stations">
   <property name="text">
    <string>Add stations </string>
   </property>
  </action>
  <action name="menuChangeLocators">
   <property name="text">
    <string>Change Locators</string>
   </property>
  </action>
  <action name="actionImportData">
   <property name="text">
    <string>Import data</string>
   </property>
  </action>
  <action name="actionDataMonitor">
   <property name="text">
    <string>Data availlability monitor</string>
   </property>
  </action>
  <action name="actionAbout_KGminer">
   <property name="text">
    <string>About KGminer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
